CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+* Having more clear or up-to-date instructions in the README for both [`brave-browser`](https://github.com/brave/brave-browser/blob/master/README.md) and [`brave-core`](https://github.com/brave/brave-core/blob/master/README.md).
+* Capturing/updating helpful information [in our wiki](https://github.com/brave/brave-browser/wiki) (you'll need to request permission)
+* Helping to propose a way to bring documentation to other languages. Right now, everything is in English
+* Improving this document :smile:
+
+### Help with translations
+All text being added to Brave is done initially in English (en-US) and then is translated by real people into other languages.
+We're missing translations for many languages and some translations might be incomplete or poor quality.
+
+For everything you'd need to get started, check out https://www.transifex.com/brave/brave/ :smile:
+
+### Work on the code
+* The [repo's wiki](https://github.com/brave/brave-browser/wiki) has instructions for cloning the repo and getting setup on your platform of choice
+* Check out the [troubleshooting page](https://github.com/brave/brave-browser/wiki/Troubleshooting) if you get stuck
+* Once you're up and running, find an interesting issue to fix. Check out issues labelled with [good first issue](https://github.com/brave/brave-browser/labels/good%20first%20issue)
+  - some issues only require knowledge of JavaScript (for example, pages using React and our [Brave UI library](https://github.com/brave/brave-ui))
+  - other issues may require C++ changes in either the Brave code or in Chromium
+
+## Getting started
+* Make sure you have a [GitHub account](https://github.com/join).
+* Submit a [ticket](https://github.com/brave/brave-browser/issues) for your issue if one does not already exist. Please include the Brave version, operating system, and steps to reproduce the issue.
+* Fork the repository on GitHub (this might be [`brave-browser`](https://github.com/brave/brave-browser), [`brave-core`](https://github.com/brave/brave-core), or both).
+* For changes to JavaScript files, we recommend installing a [Standard](http://standardjs.com/) plugin for your preferred text editor in order to ensure code style consistency.
+* For C++ changes, you can consider setting up [clang-format](https://chromium.googlesource.com/chromium/src/+/master/docs/sublime_ide.md#Format-Selection-with-Clang_Format-Chromium-only) for your editor.
+* For changes which involve patches, please check out our [Patching Chromium](https://github.com/brave/brave-browser/wiki/Patching-Chromium) guide.
+
+### Making changes
+Once you've cloned the repo to your computer, you're ready to start making edits!
+
+Please note that there are two repositories here:
+* the root project ([`brave-browser`](https://github.com/brave/brave-browser)), which pulls down all of the Chromium code into `src/`
+* [`brave-core`](https://github.com/brave/brave-core) which is located on disk under the root at `src/brave`
```

Comment:
```suggestion
* [`brave-core`](https://github.com/brave/brave-core) which is located on disk under the root at `src/brave`. This repo is a sub-repo of brave-browser and you don't need to pull it separately. This is where the core changes of Brave lives, including key functionalities such as Rewards and Shields. It is also where all extensions and webUI interfaces are located.
```

Reply:
not necessarily that text but I think it's important to say that brave-core is a sub-repo of brave-browser and that's where the code is located for most changes a contributor can make